Resources/Textures/Objects/Weapons/Throwable/bola.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "license": "",
+    "copyright": "",
```

Comment:
Taking the above example, if you head to the clothing/head.dmi in the icons section of the github you'll see something along the lines of "Latest commit f095264 on 26 Oct 2021" on the centre right beside the history button. If you right click the commit f095264 and copy link, that will get you the commit link.

Then stick that in this copyright section with "taken from [codebase name] at commit [insert commit link you just got]"

Hope that helps.

Reply:
So does this have to be the specific link to the commit that ported bola's over to the codebase I'm looking at or whatever just the latest commit is that touched that file?  I have the commit list but I can't find a specific one for bolas.